packages/jest-util/src/__tests__/installCommonGlobals.test.ts

```diff
@@ -7,21 +7,28 @@
 
 import {createContext, runInContext} from 'vm';
 
+declare global {
+  function DTRACE_NET_SERVER_CONNECTION(): unknown;
+}
+
+const fake = jest.fn();
+globalThis.DTRACE_NET_SERVER_CONNECTION = fake;
+
 let installCommonGlobals: typeof import('../installCommonGlobals').default;
-let fake: jest.Mock;
 
 function getGlobal(): typeof globalThis {
   return runInContext('this', createContext());
 }
 
 beforeEach(() => {
-  fake = jest.fn();
-  // @ts-expect-error
-  globalThis.DTRACE_NET_SERVER_CONNECTION = fake;
-
   installCommonGlobals = require('../installCommonGlobals').default;
 });
 
+afterEach(() => {
+  jest.clearAllMocks();
+  jest.resetModules();
+});
```

Comment:
Tricky one. If I got it right: `fake` was being reset manually, looks like `jest.clearAllMocks()` does the same job; also `require` in `beforeEach` above has no effect without `jest.resetModules()` because of module caching. Or?